lib/matplotlib/tests/test_figure.py

```diff
@@ -25,7 +25,7 @@
 import matplotlib.dates as mdates
 
 
-@image_comparison(['figure_align_labels'], extensions=['png', 'svg'],
+@image_comparison(['figure_align_labels.png'],
```

Comment:
Maybe this one should stay.  @anntzer 's comments in 19359c11c5778b625d0e9e6b8a26f04655342830 when the pdf version was removed suggest that this was testing some of the fixed-dpi issues we have with vector formats.

Reply:
Sure, can revert. That commit also has some backcompat stuff that maybe could be removed in #29816, looks like...